README.md

```diff
@@ -44,6 +44,8 @@ npm install -g create-react-app
 
 **We strongly recommend to use Node >= 6 and npm >= 3 for faster installation speed and better disk usage.** You can use [nvm](https://github.com/creationix/nvm#usage) to easily switch Node versions between different projects.
 
+**Note:** `nvm` does not support Windows (nvm issue [#284](https://github.com/creationix/nvm/issues/284)). See potential [alternatives](https://github.com/creationix/nvm#installation).
```

Comment:
Can you embed this into previous sentence please? It's not super essential and I don't want to clobber up the page. For example "you can use nvm or alternatives to ..."